test/test_oneview_logical_switch_group.py

```diff
@@ -165,15 +170,18 @@ def test_update_scopes_when_different(self):
 
         different_resource = params_to_scope['data'].copy()
         different_resource['scopeUris'] = ['fake']
-        self.resource.get_by.return_value = [different_resource]
-        self.mock_ov_client.switch_types.get_by.return_value = [{'uri': SWITCH_TYPE_URI}]
+        self.resource.data = different_resource
+        obj = mock.Mock()
+        obj.data = {'uri': SWITCH_TYPE_URI}
+        self.mock_ov_client.switch_types.get_by_name.return_value = obj
 
-        self.resource.patch.return_value = params_to_scope['data']
+        obj1 = mock.Mock()
+        obj1.data = params_to_scope['data']
+        self.resource.patch.return_value = obj1
 
         LogicalSwitchGroupModule().run()
 
-        self.resource.patch.assert_called_once_with('rest/fw/fake',
-                                                    operation='replace',
+        self.resource.patch.assert_called_once_with(operation='replace',
```

Comment:
Is that  uri  not required here ?

Reply:
no, its not required, as the operation is calling on an object of the resource.